pkg/registry/pod/rest/log.go

```diff
@@ -58,6 +58,9 @@ func (r *LogREST) Get(ctx api.Context, name string, opts runtime.Object) (runtim
 	if err != nil {
 		return nil, err
 	}
+	if location.Host == "" {
```

Comment:
did you start seeing an empty host here with this PR, or was this precautionary?